.github/workflows/ci.yml

```diff
@@ -35,6 +30,8 @@ jobs:
         run: |
           docker-compose build
           docker images
+          docker tag navigator_backend navigator-backend
```

Comment:
@joel-wright / @henry-cpr ... I think this is the only point of contention - happy to change the name of the ECR registries to have underscores (not hyphens) - but my preference would be to change the `docker-compose.yml`

Reply:
Can you take me through the either/or thinking and what changes are involved?